### ISSUE TITLE
How to training detector and Siamese?

### ISSUE THREAD

**Issue by ray114115** [phil-bergmann/tracking_wo_bnw#6]
Hi, If I want to train object detector, I need to convert my dataset format from MOT17 to VOC, is that right? or I don't need to do that?
How about training the re-identifaction Siamese network? What kind of format should be? MOT17 or VOC format?

**Comment by timmeinhardt** [comment]
Is this related to #4? The FPN object detector can be trained on MOT17 and VOC. Our code includes dataset classes for both classes.
In general, you do not convert datasets in PyTorch. You create your own dataset class in `src/fpn/fpn/datasets` and adjust it to the structure of your dataset. Please have look at the existing datasets, make yourself familiar with the code structure and then implement your dataset class for training.

**Comment by ray114115** [comment]
Yes, this is related to #4. Because that issue is closed, I reopen a new one.

I have successfully trained the detector now. Thank you very much, you have helped me a lot. I will try Siamese after finish the training on detector.

**Comment by ray114115** [comment]
Hi, I trained the Siamese. How can I test the Siamese network?

**Comment by timmeinhardt** [comment]
There is no testing for the Siamese network. The final goal of the Siamese network is to provide a similarity score of image patches for re-identification. The validation loss allows to monitor the quality of that score during the training. If the network trained successfully, i.e., the validation loss decreased without overfitting, the Siamese network can be tested by applying it for re-identification.

**Comment by timmeinhardt** [comment]
Can we close this issue?
